java-vertexai/google-cloud-vertexai/src/main/java/com/google/cloud/vertexai/generativeai/GenerativeModel.java

```diff
@@ -114,20 +118,22 @@ public static class Builder {
     private GenerationConfig generationConfig = GenerationConfig.getDefaultInstance();
     private ImmutableList<SafetySetting> safetySettings = ImmutableList.of();
     private ImmutableList<Tool> tools = ImmutableList.of();
+    private Content systemInstructions = null;
```

Comment:
According to Java readability reviewers, using `null` is discouraged. Can we switch to `Content.getDefaultInstance()` or `Optional`?

Reply:
I'll go with `Optional` as it's clearer when system instructions have been set or not (otherwise I'd have to examine the `Content` instance to see if it's a default one or a normal one).
I think it'll be clearer / cleaner that way.